### ISSUE TITLE
Support connecting to TLS 1.3 only hosts

### ISSUE THREAD

**Issue by sugar700** [babelmark/babelmark-proxy#2]
While checking pulldown-cmark I get the following error:

> The SSL connection could not be established, see inner exception.
> Authentication failed because the remote party sent a TLS alert: 'ProtocolVersion'.
> The message received was unexpected or badly formatted.

This is because the host is TLS 1.3 only. Would it be possible to update .NET so that it could connect to TLS 1.3 only hosts. I checked `babelmark-proxy` locally and it can connect to TLS 1.3 fine, so I assume the issue is the version of .NET on the server used by babelmark.

**Comment by xoofx** [comment]
Indeed, I don't know what is happening. The server should run the same version but there is something different with the `HttpClient` and Ssl protocols for some reasons. Tried a couple of tweaks without much success for now.

**Comment by SilverBut** [comment]
This might due to OS version... is there any way people can reproduce it in a virtual machine?

**Comment by xoofx** [comment]
Closing as I don't think the issue is here anymore, though all Rust are down - but it looks like its their server.
